### PR TITLE
Update V2.0.0 correlation section

### PR DESCRIPTION
**Pull request by frack113**
Warning This PR is only for the Correlation rules section.

The meta-rule-schema.json file in progress 

- Small text cleanup 
- Remove useless ordered boolean
- Remove `generate` move to [discussion](https://github.com/SigmaHQ/sigma-specification/discussions/113)
- discussion [V2.0.0 Meta-rule Time Selection ](https://github.com/SigmaHQ/sigma-specification/discussions/116)

### CODE REVIEW

File: appendix_meta_rules.md [hunk 1]
```diff
@@ -135,6 +178,7 @@ Like sigma rules , correlation rules have a title and a unique id to identify th
 **Attribute:** title
 
 A brief title for the rule that should contain what the rule is supposed to detect (max. 256 characters)
+<!-- shouldn't it be mandatory ?-->
```
Comment (phantinuss) [inline appendix_meta_rules.md:181]: It is. That is defined in the schema. At least for the existing spec, it wasn't defined in the text body if a key is required if it's optional.

File: appendix_meta_rules.md [hunk 2]
```diff
@@ -104,6 +104,49 @@ Example: A valid GPO script that triggers multiple Sigma rules.
 
 All rules in a file, basic event rules as well as correlations, might contain an additional attribute called "generate".
 If it is set to true, the rule will generate a query, even if it is referred to by other correlations. Otherwise by default no "standalone" query would be generated for this rule.
```
Comment (phantinuss) [inline appendix_meta_rules.md:106]:
```suggestion
The rules in a multi-document YAML that build a correlation rule are not producing single, independent queries. They are used as a tool to define more complex constructs out of basic Sigma detections. Therefore only the outermost correlation rule may define meta information such as status, level, date or anything else.
```

It's pretty much under defined like this but the spirit of correlation rules should be more clear like that.

A question I asked myself is, if we even need to define the use case "subquery should generate it's own query" as you can just duplicate it as it's own rule. At least that's what I would suggest.

**Comment by Res260** [comment]
Hello @frack113,  I'm aware that this is a draft PR, but after reading its content, I feel the need to ask: has a choice been made yet regarding generate/silent/default to informational?

If not: please consider not going the "default to informational" route. Defining level behaviors in the specification is useful for a common rule repository like sigmahq/sigma, but makes less sense to apply it to every sigma users. 

Internally, we have two types of rules: indicators and normal rules. Indicators are used for contextualization of assets and are correlated together to risk-score assets. The normal rule trigger individual alerts.
With the "default to informational" proposal, it would make for a very awkward solution, in my opinion.

Sorry if this is not the place for such comment, I didn't know where to write it.
Thanks for your work, can't wait for correlations! :)

**Comment by phantinuss** [comment]
> Hello @frack113, I'm aware that this is a draft PR, but after reading its content, I feel the need to ask: has a choice been made yet regarding generate/silent/default to informational?
> 
> If not: please consider not going the "default to informational" route. Defining level behaviors in the specification is useful for a common rule repository like sigmahq/sigma, but makes less sense to apply it to every sigma users.
> 
> Internally, we have two types of rules: indicators and normal rules. Indicators are used for contextualization of assets and are correlated together to risk-score assets. The normal rule trigger individual alerts. With the "default to informational" proposal, it would make for a very awkward solution, in my opinion.
> 
> Sorry if this is not the place for such comment, I didn't know where to write it. Thanks for your work, can't wait for correlations! :)

@Res260: Thanks for your input. We came also to that conclusion in internal discussion and it definitely won't be pinned to the level of a rule. The rest is still open to discussion though.

File: appendix_meta_rules.md [hunk 3]
```diff
@@ -102,8 +102,50 @@ Example: A valid GPO script that triggers multiple Sigma rules.
...
+My proposal is to define it like this:
+
+    We can switch back to silent is false by default (implicitly). And silent is true if you set it to true explicitly or if there is no level defined in a rule, which is the case for all "in-file/multi-document" correlation rules that are only used as aggregates.
+
+ phantinuss Dec 14, 2023
+
+Another idea how we could specify it:
+If a rule has no level set, the default level is "informational". That also should work and we have no need to define either "generate" or "silent" at all.
+
+-->
```
Comment (phantinuss) [inline appendix_meta_rules.md:148]:
```suggestion
```

**Pull request merged**